Example/AblyChatExample/Mocks/Misc.swift

```diff
@@ -37,6 +37,10 @@ final class MockMessagesPaginatedResult: PaginatedResult {
         self.roomID = roomID
         self.numberOfMockMessages = numberOfMockMessages
     }
+
+    static func == (_: MockMessagesPaginatedResult, _: MockMessagesPaginatedResult) -> Bool {
+        fatalError("Not implemented")
```

Comment:
Why bother to add it here then?

Reply:
`PaginatedResult` conforms to `Equatable` so this implementation of it also needs this